src/components/notifications/PageNotifications.vue

```diff
@@ -75,8 +110,11 @@ export default {
         }
       },
       /* istanbul ignore next */
-      update(data) {
-        return data.notifications
+      update(result) {
+        this.dataLoaded = true
+        // assume that when the full page got loaded, that there is more
+        this.moreAvailable = (result.notifications.length % 20 === 0)
+        return result.notifications
```

Comment:
OK, I was debugging Marmot Day and I see that here it is always returning the same 20 transaction O.o

Debugging more

Reply:
Probably an API thing. Maybe timestamp is not handled correctly

Reply:
I tried sending the latest timestamp of the 20 notifications to the API (in the playground) to get the next 20 notifications which was working fine.